osu.Game/Online/Multiplayer/MultiplayerClient.cs

```diff
@@ -143,7 +143,29 @@ public virtual bool IsHost
         [Resolved]
         private UserLookupCache userLookupCache { get; set; } = null!;
 
-        protected Room? APIRoom { get; private set; }
+        private Room? apiRoom;
+
+        protected Room? APIRoom
+        {
+            get => apiRoom;
+
+            private set
+            {
+                apiRoom = value;
+
+                NoScoreMultiplier.UnbindBindings();
+
+                if (apiRoom == null)
+                {
+                    NoScoreMultiplier.SetDefault();
+                    return;
+                }
+
+                NoScoreMultiplier.BindTarget = apiRoom.NoScoreMultiplier;
+            }
+        }
+
+        public Bindable<bool> NoScoreMultiplier = new Bindable<bool>(false);
```

Comment:
This still baffles me. The bindable is essentially global state for a given client instance. Why does it even need to exist in this way? Why does a communication component concern itself with scoring? The _consumers_ of the communication component - namely `MultiplayerPlayer` - should be the one caring about scoring.